alacritty_config_derive/tests/config.rs

```diff
@@ -124,18 +124,20 @@ fn config_deserialize() {
     assert_eq!(test.flatten.flatty, 123);
 
     // Verify all log messages are correct.
-    let error_logs = logger.error_logs.lock().unwrap();
+    let mut error_logs = logger.error_logs.lock().unwrap();
+    error_logs.sort_unstable();
     assert_eq!(error_logs.as_slice(), [
         "Config error: enom_error: unknown variant `HugaBuga`, expected one of `One`, `Two`, \
          `Three`",
         "Config error: field1: invalid type: string \"testing\", expected usize",
     ]);
-    let warn_logs = logger.warn_logs.lock().unwrap();
+    let mut warn_logs = logger.warn_logs.lock().unwrap();
+    warn_logs.sort_unstable();
     assert_eq!(warn_logs.as_slice(), [
-        "Config warning: field1 has been deprecated; use field2 instead\nUse `alacritty migrate` \
-         to automatically resolve it",
         "Config warning: enom_error has been deprecated\nUse `alacritty migrate` to automatically \
          resolve it",
+        "Config warning: field1 has been deprecated; use field2 instead\nUse `alacritty migrate` \
+         to automatically resolve it",
```

Comment:
This actually failed after the update, it should never fail again with the sort in place. I don't think order actually matters so this should be fine.